generate-sandpack-config.mjs

```diff
@@ -0,0 +1,118 @@
+import _ from 'lodash';
+
+import fs from 'fs';
+import path from 'path';
+
+const examples = ['banner'];
+
+console.log('Sandpack config factory 🏭');
+
+examples.forEach(example => {
+  let __dirname = path.join(example);
```

Comment:
what do the two underscores mean and can it be a const declaration?

Reply:
__dirname is normally a predefined node thing but If you are using Node.js modules __dirname and __filename don't exist. so just using that naming scheme